lib/ansible/utils/module_docs_fragments/oneview.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+class ModuleDocFragment(object):
+
+    # OneView doc fragment
+    DOCUMENTATION = '''
+options:
+    config:
+      description:
+        - Path to a .json configuration file containing the OneView client configuration.
+          The configuration file is optional. If the file path is not provided, the configuration will be loaded from
+          environment variables.
+      required: false
+
+notes:
+    - "A sample configuration file for the config parameter can be found at:
+       U(https://github.com/HewlettPackard/oneview-ansible/blob/master/examples/oneview_config-rename.json)"
+    - "Check how to use environment variables for configuration at:
+       U(https://github.com/HewlettPackard/oneview-ansible#environment-variables)"
+    - "Additional Playbooks for the HPE OneView Ansible modules can be found at:
+       U(https://github.com/HewlettPackard/oneview-ansible/tree/master/examples)"
+    '''
+
+    VALIDATEETAG = '''
```

Comment:
Hello, it is apparently able to use it when extending documentation.

I see this inside the FCNetworkModule:
```
extends_documentation_fragment:
    - oneview
    - oneview.validateetag
```

And looking at the `oneview_fc_network_module.rst` file inside `docs`:

```
(...)
        List with the Fibre Channel Network properties.        
                state<div style="font-size: small;">
    no
    
        presentabsent
        Indicates the desired state for the Fibre Channel Network resource. <code>present</code> will ensure data properties are compliant with OneView. <code>absent</code> will remove the resource from OneView, if it exists.        
                validate_etag<div style="font-size: small;">
    no
    True
        truefalse
        When the ETag Validation is enabled, the request will be conditionally processed only if the current ETag for the resource matches the ETag provided in the data.        
        
    

(...)
```
And we also have facts modules which call the FACTSPARAMS for extending documentation.